spharpy/samplings/samplings.py

```diff
@@ -599,18 +599,17 @@ def gaussian(n_points=None, n_max=None, radius=1.):
 def eigenmike_em32():
     """Microphone positions of the Eigenmike em32 by mhacoustics.
 
-    The data are according to the Eigenstudio user manual on the homepage [#]_.
-
-    References
-    ----------
-    .. [#]  Eigenstudio User Manual, https://mhacoustics.com/download
-
+    The data are according to the Eigenmike user manual on the homepage [#]_.
```

Comment:
```suggestion
    The data are according to the Eigenstudio user manual on the homepage [#]_.
```
This was correct before.